templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -52,7 +52,11 @@ jobs:
           git checkout {{ plugin_default_branch }}
           bump2version --no-commit minor
 
-      - name: Make a PR with version bump
+      - name: Remove entries from CHANGES directory
+        run: |
+          git rm CHANGES/*
```

Comment:
I think this also deletes the dot files and the CHANGES folder itself.

Reply:
Have you tested the command on your machine? On mine in an 'empty' plugin CHANGES directory it deleted both dot files and the directory itself. Maybe this isn't a fair comparison if we expect there to always be changelog entries present.

```
[gerrod@localhost pulp_python]$ git rm CHANGES/*
rm 'CHANGES/.TEMPLATE.rst'
rm 'CHANGES/.gitignore'
[gerrod@localhost pulp_python]$ ls -la CHANGES/
ls: cannot access 'CHANGES/': No such file or directory
```

Reply:
Your shell has globstar enabled. I'll improve the patch to be more specific and only remove entries.